tests/apps/courses/test_models_course_run.py

```diff
@@ -22,6 +22,16 @@ def setUp(self):
         super().setUp()
         self.now = timezone.now()
 
+    def test_models_course_run_state_coming_soon(self):
+        """
+        A course run that has no value passed should return a state with 
```

Comment:
```suggestion
        A course run that has no start date should return a state with 
```

Reply:
This has been commited. I'll push it after the other changes.